src/ImgixTag.js

```diff
@@ -187,14 +187,16 @@ var ImgixTag = (function () {
 
   ImgixTag.prototype.sizes = function () {
     var existingSizes = this.el.getAttribute('sizes');
+    var ixSizes = this.el.getAttribute('ix-sizes');
     const el = this.el;
     const _window = this.window;
 
-    if (existingSizes && existingSizes !== 'auto') {
-      return existingSizes;
-    } else if (existingSizes === 'auto') {
-      return autoSize.updateOnResize({ el, existingSizes, _window });
+    if (existingSizes || ixSizes !== 'auto') {
+      return existingSizes ? existingSizes : '100vw';
```

Comment:
What is the expected behaviour when `ix-sizes` is not set to `auto`? Maybe we should use the `ix-sizes` value as an extra fallback?
```suggestion
      return existingSizes != null ? existingSizes : (ixSizes != null ? ixSizes : '100vw');
```

FYI writing `existingSizes ? existingSizes : '100vw'` is the same as `existingSizes || '100vw'`, but you probably want something like `existingSizes != null ? existingSizes : '100vw'` to handle falsy values.

Reply:
On second thought, what's the use case for allowing anything other than `auto` as a value `ix-sizes`? Wouldn't in that case we rather force the user to make use of the built in `sizes` attribute?

I think it's closer to user expectations to have `ix-sizes` onyl accept `auto` as a value, and to log an warning if any other value is passed in.

Reply:
It's not so much a use case as much as "if it's possible, _someone_ is going to do it". What I mean by this is since we're giving people this `ix-sizes` attribute, it's bound to be misused. In these cases I always prefer to handle these "undocumented" cases if they can be handled logically. In this case, that would mean setting `sizes` from `ix-sizes` if it is set to a value that's not `auto`.

This is the same reason why I always advocate for reducing API surface area/reducing possible props/config options, because if you give someone the _option_ to set something to an "incorrect" value, eventually it's going to happen, and they're going to feel a bit confused when it doesn't do what they expect, even if you show them a warning.

---

Aside: technically "a use case" should be "an use case" but it doesn't sound right... English is weird

Reply:
> setting sizes from ix-sizes if it is set to a value that's not auto.

I understand better what you meant now thanks! I can definitely get behind this change as a fallback when `sizes` is not set and `ix-sizes` is not `auto`.

I'll PR this change 😉